types/backbone/index.d.ts

```diff
@@ -201,7 +214,7 @@ declare namespace Backbone {
         sync(...arg: any[]): JQueryXHR;
     }
 
-    class Model<T = any, S = Backbone.ModelSetOptions> extends ModelBase implements Events {
+    class Model<T = any, S = Backbone.ModelSetOptions, E = {}> extends ModelBase implements Events {
```

Comment:
Since this is going to break existing code, I suggest adding a comment above the `Model` class to explain to users that they can accept additional constructor options by listing them in the `E` interface.